emission/storage/decorations/common_place_queries.py

```diff
@@ -0,0 +1,78 @@
+import emission.core.wrapper.common_place as ecwcp
+import emission.core.get_database as edb
+import emission.simulation.markov_model_counter as esmmc
+import emission.storage.decorations.common_trip_queries as esdctp
+
+#################################################################################
+############################ database functions #################################
+#################################################################################
+
+def save_common_place(common_place):
+    db = edb.get_common_place_db()
+    db.save(common_place)
+
+def get_common_place_from_db(_id):
+    db = edb.get_common_place_db()
+    json_obj = db.find_one({"_id" : _id})
+    return make_common_place_from_json(json_obj)
+
+def get_all_common_places_for_user(user_id):
+    db = edb.get_common_place_db()
+    return db.find({"user_id" : user_id})
+
+def make_common_place_from_json(json_obj):
+    return ecwcp.CommonPlace(json_obj)
+
+def make_new_common_place(user_id, coords, successors):
+    props = {
+        "user_id" : user_id,
+        "coords" : coords,
+        "common_place_id" : "%s%s" % (user_id, coords["coordinates"]),
+        "successors" : successors
+    }
+    return ecwcp.CommonPlace(props)
+
+def make_common_place(props):
```

Comment:
Oh yeah youre right they are the same, we only need one

On Sun, Feb 7, 2016 at 11:01 PM, shankari <notifications@github.com> wrote:

> In emission/storage/decorations/common_place_queries.py
> <https://github.com/e-mission/e-mission-server/pull/227#discussion_r52137331>
> :
>
> > +    db = edb.get_common_place_db()
> > +    return db.find({"user_id" : user_id})
> > +
> > +def make_common_place_from_json(json_obj):
> > +    return ecwcp.CommonPlace(json_obj)
> > +
> > +def make_new_common_place(user_id, coords, successors):
> > +    props = {
> > +        "user_id" : user_id,
> > +        "coords" : coords,
> > +        "common_place_id" : "%s%s" % (user_id, coords["coordinates"]),
> > +        "successors" : successors
> > +    }
> > +    return ecwcp.CommonPlace(props)
> > +
> > +def make_common_place(props):
>
> I understand what make_new_common_place is. My comment is for
> make_common_place.
> What is the difference between make_common_place and
> make_common_place_from_json?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/227/files#r52137331>.
>